google/cloud/spanner_v1/_opentelemetry_tracing.py

```diff
@@ -80,7 +80,7 @@ def trace_call(name, session, extra_attributes=None, observability_options=None)
     attributes = {
         "db.type": "spanner",
         "db.url": SpannerClient.DEFAULT_ENDPOINT,
-        "db.instance": session._database.name,
+        "db.instance": "" if not session._database else session._database.name,
```

Comment:
Please see comment below for more details.
Can we get this value from observability_options in case session is. not passed?

Reply:
Yes indeed.